main.py

```diff
@@ -37,6 +38,7 @@ def setup(self):
         self._switches_lock = Lock()
         self._flow_manager_url = settings.FLOW_MANAGER_URL
         self._color_field = settings.COLOR_FIELD
+        self.table_id = 0
```

Comment:
This works, as for NApps like coloring that only have a "base" table group this suffices, but reading the code here notice that even the table group doesn't become evident until you read how the event is being published. 

A pattern that you can use on all NApps regardless of number of table groups, would be to have `self.table_groups: dict[str, int]`, where it map which which table a group is using, which would be updated based on `"kytos/of_multi_table.enable_table"`. Then you could initialize this with `self.table_groups = {"base": 0}`, and then introduce a method `def set_flow_table_group_owner(self, flow: dict) -> dict` that given a flow dict it sets both `table_group` and `owner` that this flow will need and derives to which table that flow should be installed, internally it could check which match, priority or anything else that uniquely identifies this given flow, which table group it should be set to and then ultimately doing a lookup on `self.table_groups[name]`, finally it returns the dict with should have its `table_group, owner, table_id` updated, that way you can pass it down to any other flow mod build funcs/methods and ultimately to flow_manager. 

Wdyt? Let me know if you see a simpler way, I think this would integrate very nicely with how mef_eline is already [identifying EVPLs and EPLs](https://github.com/kytos-ng/mef_eline/blob/master/models/evc.py#L1062)

Reply:
Oh yeah, I totally forgot about `owner` and `table_group`. I am going to use `table_group` as classification of the flow. Added `group` as an str argument to `set_flow_table_group_owner`.